lib/ae_mdw/db/name.ex

```diff
@@ -186,6 +187,16 @@ defmodule AeMdw.Db.Name do
             name_hash == :aens_transfer_tx.name_hash(tx)
           end)
           |> :aens_transfer_tx.account_id()
+
+        {_block_hash, tx_type, _signed_tx, tx_rec}
+        when tx_type in [:ga_meta_tx, :paying_for_tx] ->
+          {_mod, tx_rec} =
```

Comment:
Can we pattern-match `:name_claim_tx` here using `:aetx.specialize_type()` at the end?

That way if the transaction belongs to the wrong type we get a pattern-match error instead of an error triggered by the `:aens_claim_tx/account_id/1`

Reply:
sure, replaced the `specialize_callback` with `specialize_type` for that.